android/AndroidManifest.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
```

Comment:
Did you mean to commit android related stuff in this branch?

Reply:
Huh, I reviewed the whole PR and didn't see that. I wonder why. I'll remove it.